request.go

```diff
@@ -356,8 +350,12 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 				} else if v.Kind() == reflect.Int {
 					at = v.Int()
 				} else {
-					er = ErrInvalidTime
-					break
+					// Return error immediately to ensure a runtime panic doesn't swallow it.
+					return &ErrorObject{
```

Comment:
I'm still having trouble with this.  To me, it is not the same to error on (de)serilization and for you to render and error response.  I would like this PR to simply introduce two structs and a method to marshal them.